### PR TITLE
admission trials

### PR DESCRIPTION
**Pull request by Jeddaa**
A pull request for review on project done during admission trials.
I am to review the project done by playing the role of a code reviewer

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+   
+    <script  src="js/index.js"></script>
+  <br>
+  </br>
+  <main id="main"> 
+  <h1 id="title">
```
Comment (Jeddaa) [inline index.html:16]: It would be better to use the header tags for the header section

File: index.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        <strong>2005</strong> - states "we will have to double the world food supply by 2050." Argues that genetically modified crops are the only way we can meet the demand, as we run out of arable land. Says that GM crops are not inherently dangerous because "we've been genetically modifying plants and animals for a long time. Long before we called it science, people were selecting the best breeds." </li>
+    
+        <li>
+          <strong>2009</strong> - dies at the age of 95.</li> 
+      </ul>
+      <blockquote>
+        <p>"Borlaug's life and achievement are testimony to the far-reaching contribution that one man's towering intellect, persistence and scientific vision can make to human peace and progress."</p>
+      <cite>-- Indian Prime Minister Manmohan Singh</cite>
+      </blockquote>
+    <p>If you have time, you should read more about this incredible human being on his<a id="tribute-link" target="_blank" href="https://en.wikipedia.org/wiki/Norman_Borlaug">Wikipedia entry</a></p>
```
Comment (Jeddaa) [inline index.html:59]: This might have been better as a section for the bottom of the page

File: index.html [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  </div>
+
+  </main>
```
Comment (Jeddaa) [inline index.html:62]: KIndly use section tags instead of main tags in your code. It aslo helps in browser support

File: index.html (hunk 2 above)
Comment (Jeddaa) [inline index.html:59]: I think this should be a section on it's own as it is probably meant to be the last thing on the page.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  <h1 id="title">
+    Dr. Norman Borlaug
+  </h1>
+  <p>The man who saved a billion lives</p>
+  <div id="img-div">
+    <img id="image" src="tribute.jpg" alt="Dr. Norman Borlaug seen standing in Mexican wheat field with a group of biologists.">
+    <figcaption id="img-caption">
+      Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.
+    </figcaption>
+  </div>
```
Comment (Jeddaa) [inline index.html:25]: Kindly use the header tag to group this as i think it is meant to be the header part of the page

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  margin: 50px 0;
+}
+
+ul {
+  max-width: 550px;
+  margin: 0 auto 50px auto;
+  text-align: left;
+  line-height: 1.6;
+}
+blockquote {
```
Comment (Jeddaa) [inline css/style.css:32]: please follow the linters rule and give a line space in between each styles

**Review by Jeddaa — COMMENTED** [review]
Kindly cross-check and sort out all raised changes. Check to see the reason why some of the linters are not checking.
Keep up the good work and good luck

**Review by TadesseDev — APPROVED** [review]
### Hi @Jeddaa 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [ ]  phrased the comments politely.✅
 - [ ]  includes an explanation, a "why". ✅
## Optional suggestions
- [Optional] It would be wise to use a couple of emojis and/or gifs in the review summary. This will make your review more friendly 
- [Optional] Try to be more detailed in your comments, explain more and provide some links that could help in resolving/understanding the issue. 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+        <strong>1953</strong> - crosses a short, sturdy dwarf breed of wheat with a high-yeidling American breed, creating a strain that responds well to fertilizer. It goes on to provide 95% of Mexico's wheat.     </li>      <li>
+        <strong>1962</strong> - Visits Delhi and brings his high-yielding strains of wheat to the Indian subcontinent in time to help mitigate mass starvation due to a rapidly expanding population  </li>
+      <li>
+        <strong>1970</strong> - receives the Nobel Peace Prize</li> 
+        <li>
+        <strong>1983</strong>- helps seven African countries dramatically increase their maize and sorghum yields   </li>      <li>
+        <strong>1984</strong>- becomes a distinguished professor at Texas A&M University   </li>      <li>
+        <strong>2005</strong> - states "we will have to double the world food supply by 2050." Argues that genetically modified crops are the only way we can meet the demand, as we run out of arable land. Says that GM crops are not inherently dangerous because "we've been genetically modifying plants and animals for a long time. Long before we called it science, people were selecting the best breeds." </li>
```
Comment (Jeddaa) [inline index.html:50]: test comment

**Pull request merged**